packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -80,3 +84,85 @@ When you upgrade an embedded Sign-In Widget:
+When [External Identity Providers (IdPs)](https://github.com/okta/okta-signin-widget#openid-connect) are used in Social Login, the supported IdPs (Google, Facebook, Apple, Microsoft, and LinkedIn) are declared with a type, get distinct styling and get default i18n text, while other entries receive general styling and require text to be provided. Each IdP can have additional CSS classes added through an optional `className` property. This feature still functions, however, it's no longer the preferred method to enable the link for users to initialize the route. This method can be replaced by configuring a Routing Rule with the application context.
+
+You no longer need the [IdPs](https://github.com/okta/okta-signin-widget#openid-connect) JavaScript object and `className` property in the Sign-In Widget, and you can remove them as follows.
+
+![Displays the IdPs object and className property in JavaScript to remove](/img/SIW_Upgrade_Config_Change3.png)
+
+### Smart card IdP
+
+You no longer need to configure the authentication settings for the smart card IdP, as it's no longer supported. You can remove the following authentication settings for the [smart card IdP](https://github.com/okta/okta-signin-widget#smart-card-idp) as follows.
+
+![Displays the smart card IdP settings in JavaScript to remove](/img/SIW_Upgrade_Config_Change4.png)
+
+### Bootstrapping from a recovery token
+
+If you're initializing the Widget with a [recovery token](https://github.com/okta/okta-signin-widget#bootstrapping-from-a-recovery-token), the `recoveryToken` setting appears, for example:
+
+![Displays the recovery token setting](/img/SIW_Upgrade_Config_Change5.png)
+
+The recovery token is dynamic and is automatically passed into the initialization of the Widget. A value in the `recoveryToken` setting currently doesn't have any effect on Widget function, though, the setting will take effect in the future.
+
+### Okta dashboard or custom dashboard sign-in
+
+For [sign-in with the Okta dashboard](https://developer.okta.com/code/javascript/okta_sign-in_widget/#sign-in-to-okta-with-the-default-dashboard), you no longer need to configure a redirect to the Okta Identity Cloud, create an Okta session, and then open a URL specified in the Widget. You can remove the redirect configuration as follows.
+
+![Displays the Okta dashboard sign-in](/img/SIW_Upgrade_Config_Change6.png)
+
+### Feature flags
+
+The only feature that is supported when you upgrade the Widget is `features.hideSignOutLinkInMFA`, which hides the sign-out link for MFA challenge.
+
+The following specific features are no longer supported, and you must remove them from `features` in the JSON code:
+
+* `features.rememberMe` &mdash; Displayed the "Remember me" function at sign-in. This feature is no longer configurable. See the [Organization Settings](https://help.okta.com/oie/en-us/Content/Topics/Security/Security_General.htm?cshid=Security_General) section in the Okta product documentation to enable this feature.
```

Comment:
Can we not have these start with active verbs rather than past tense? Displays, Recovers...and so on?

Also - this is an "in product" alias to the product doc page, not an external alias....is there an external alias available?

```suggestion
* `features.rememberMe` &mdash; Displayed the "Remember me" function when a user signs in. This feature is no longer configurable. See the [Organization Settings](https://help.okta.com/oie/en-us/Content/Topics/Security/Security_General.htm?cshid=Security_General) section in the Okta product documentation to enable this feature.
```